db/migrate/20190116150033_add_protocols_participants_id_to_appointments.rb

```diff
@@ -0,0 +1,19 @@
+class AddProtocolsParticipantsIdToAppointments < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
This should be an `up` method. Because of the one-way data changes it will break on rollback.